mkdocs.yml

```diff
@@ -13,7 +13,6 @@ nav:
       - Introduction: "structure.md"
       - Getting started: "setup.md"
       - "deploy.md"
-      - Tests: "tests.md"
```

Comment:
Should we add `sda-dev-test-doc.md` here instead?

Reply:
but we will have it guides and in structure or just structure?